packages/math/src/IPoint.ts

```diff
@@ -0,0 +1,62 @@
+export interface IPoint
+{
+    x: number;
+    y: number;
+    copyFrom(p: IPoint): this;
+    copyTo(p: IPoint): IPoint;
```

Comment:
An improvement for this method would be `copyTo<T extends IPoint>(p: T): T;`

Reply:
ok, i can do it, but im after to put it in jsdocs :)